### ISSUE TITLE
vaxstation3100m76 spontaneous reboot?

### ISSUE THREAD

**Issue by TheFausap** [simh/simh#688]
Hello,
I was able to install OpenVMS 7.1 under vaxstation3100m76 with the following configuration:
```
set cpu 32m
set cpu idle=vms
set cpu conhalt
att nvr nvram.bin
att xs eth0
set rz0 rz26l
set rz4 cdrom
set rz5 tz30
att rz0 disk0.img
att rz4 VAXVMS071.iso
att rz5 tz30.tap
att dz 3100
```
I logged out, then during the night, I suppose, the system rebooted and now I have this on the console
```
?54 RETRY
?54 RETRY
?54 RETRY
?54 RETRY
?54 RETRY
?54 RETRY
?54 RETRY
?54 RETRY
?54 RETRY
?54 RETRY
?54 RETRY
```
If I remember well this error happens when the system didn't find a MOP server, during the boot.

After rebooted the cpu, the system is fine.

**Comment by agn453** [comment]
Have you set the console BOOT variable?  I seem to remember if it isn't set then the system defaults to an Ethernet boot using MOP.  Try 'set boot dka0' at the chevron '>>>' prompt to set this into the NVRAM.

Also note that OpenVMS VAX V7.1 may require a number of updates.  I haven't kept copies of these for V7.1, but I do have them for V5.5-2, V6.2 and V7.3 (including the Y2K ones).  If you wish to try one of these versions and need them, get back to me.

**Comment by TheFausap** [comment]
Yes, thanks, I now remember that, at the time of the issue, I forgot to set the console BOOT variable, so that's the reason.
Unfortunately, at the moment, I have only OpenVMS 7.1 available, and I don't know if I can found any patch for that version.

**Comment by TheFausap** [comment]
I found the ISO of VMS7.3 !
Now I need to remember where I have the patches
